### PR TITLE
Assignment done

### PR DESCRIPTION
**Pull request by Shoili06**

### CODE REVIEW

**Comment by samrat19** [comment]
print(num1.toFloat()/num2.toFloat())

can it be better if we take the first two numbers as float then we dont need to use toFloat

check this code


import java.util.*

fun main(){

    val reader = Scanner(System.`in`)
    print("Enter teo nos : ")

    val a = reader.nextFloat()
    val b = reader.nextFloat()

    print("Enter an operator(+,_,*,/) : ")
    val choice = reader.next()!!

    val result:Float = when(choice){
        "+" -> a + b
        "-"-> a - b
        "*" -> a * b
        "/" -> a / b
        else -> 0.0F
    }

    println("$a $choice $b = $result")
}

and reduce your code size , take result = whe(){ /* your code */}

File: src/class1kot.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    var num2:Int
+    var func:String
+    val input = Scanner(System.`in`)
+    print("Enter first no. : ")
+    num1 = input.nextInt()
+    print("Enter second no. : ")
+    num2 = input.nextInt()
+    print("Enter the operator : \n1.Addition(+)\n2.Subtraction(-)\n3.Multiplication(*)\n4.Division(/)\n5.Modulus(%)\n")
+    func = input.next()
+    when(func){
```
Comment (samrat19) [inline src/class1kot.kt]: make the code beautiful

File: src/class1kot.kt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        "+" -> print(num1+num2)
+        "-" -> print(num1-num2)
+        "*" -> print(num1*num2)
+        "/" -> print(num1.toFloat()/num2.toFloat())
+        "%" -> print(num1%num2)
```
Comment (samrat19) [inline src/class1kot.kt]: the print statements can be more user-friendly, start thinking

File: src/class1kot.kt [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import java.util.*
+
+fun main(){
+
+    var num1:Int
+    var num2:Int
+    var func:String
+    val input = Scanner(System.`in`)
+    print("Enter first no. : ")
+    num1 = input.nextInt()
```
Comment (samrat19) [inline src/class1kot.kt]: you could take as val num1 = input.nextInt(), may be it can work, check once

**Review by samrat19 — COMMENTED** [review]
check the suggestions

File: src/class1kot.kt (hunk 1 above)
Comment (Shoili06) [reply src/class1kot.kt]: Yeah...will do it

**Comment by samrat19** [comment]
assignment done.

**Pull request closed without merging**